.github/workflows/smoke-test-daily.yml

```diff
@@ -55,7 +55,8 @@ jobs:
               working-directory: plugins/woocommerce
               env:
                   E2E_MAX_FAILURES: 25
-              run: pnpm exec playwright test --config=tests/e2e-pw/daily.playwright.config.js
+                  RESET_SITE: true
```

Comment:
During the global setup phase, I added some commands there that would clear the daily smoke test site (and whichever test site the e2e tests will be run against) of all pre-existing orders, products, coupons, etc. This wipe-out greatly reduces the flakiness in our E2E tests.

But since this is a destructive action, I implemented a `RESET_SITE` environment variable here that would serve as a toggle to whether the users of our E2E would indeed like to clear all these pre-existing test data or not.